common/src/main/java/tachyon/collections/IndexedSet.java

```diff
@@ -74,14 +74,14 @@
  * First, define the fields to be indexed:
  * <pre>
```

Comment:
Instead of `<pre>` and `{@literal ...}`,  wrap all code blocks in this JavaDoc in `{@code ...}`.